hazelcast/src/test/java/com/hazelcast/instance/NodeEngineThreadLeakTest.java

```diff
@@ -38,18 +24,31 @@
 import com.hazelcast.spi.impl.eventservice.impl.EventServiceImpl;
 import com.hazelcast.spi.impl.executionservice.impl.ExecutionServiceImpl;
 import com.hazelcast.spi.impl.operationparker.impl.OperationParkerImpl;
+import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Set;
+
+import static classloading.ThreadLeakTestUtils.assertHazelcastThreadShutdown;
+import static classloading.ThreadLeakTestUtils.getThreads;
+import static org.junit.Assert.fail;
 
 /**
- * Tests that exception in Node constructor leads to properly finished services.
+ * Tests that an exception in the {@link NodeEngineImpl} constructor leads to properly finished services.
```

Comment:
The problem is/was in both constructors, could you use sth like `"the {@link NodeEngineImpl} and  {@link Node} constructors"`?

Reply:
Oh, I didn't see the issues in `Node`. Fixed.